Src/WitsmlExplorer.Api/Query/WellUpdateQueries.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using Witsml.Data;
+using WitsmlExplorer.Api.Models;
+
+namespace WitsmlExplorer.Api.Query
+{
+    public static class WellUpdateQueries
```

Comment:
Should this code be in WellQueries.cs?

Reply:
WitsmlExplorer.Api.Models.Well is DTO which can't be referenced in Witsml project. Hence this class is in WitsmlExplorer.Api project.

Reply:
To update all the properties of the well, we might need to pass all the properties of Well as parameters for WellQueries.UpdateQuery which will like this

WellQueries.UpdateQuery(wellId, wellName, field, country, timezone, operator, ...)

Thought better approach is pass Well DTO and create query.

Reply:
I have reviewed more closely now and I agree with Chandra's implementation. Previously there were only two string parameters passed, which didn't have a need for a separate DTO, but now we are passing a full Well object.

I think in the future a similar approach will be necessary for the other workers as, if we want to modify more parameters than what are currently supported.

Reply:
For now following same approach as DeleteLogObjectsWorker. Will improve further when needed as we progress